### ISSUE TITLE
Boss fetch

### ISSUE THREAD

**Issue by diahuci** [dkirkby/bossdata#72]
Hi. I tried the executable script: bossfetch --verbose qso.dat but it gives me a long traceback and an IOerror. Is there something I need to setup before I put that into the command line in the terminal or do I just have to download that file first? I already have bossdata. 

The traceback goes to: /home/diane/anaconda/bin/bossfetch, line 236, in <module> main(), then the astropy.table.Table and a lot more. (I would have posted a photo, but I'm using ubuntu and am new to their GIMP program.) 

Long story short, it gives an input/output error:
 IOError: [Errno 2] No such file or directory: 'qso.dat'. I'm assuming that the fetch should work.

**Comment by dkirkby** [comment]
It sounds like you might not yet have a valid `qso.dat` file.  Try running this command to produce one:

```
bossquery -where 'CLASS="QSO" and ZWARNING=0' --max-rows 100 --save qso.dat
```

If that is still not working, please post the full traceback.

**Comment by diahuci** [comment]
The code you provides works! (Note: Needs a small syntax correction "-where" should be "--where".)
